tests/component/jhi-item-count.component.spec.ts

```diff
@@ -22,22 +22,14 @@ import { JhiItemCountComponent } from '../../src/component/jhi-item-count.compon
 import { JhiTranslateComponent } from '../../src/language/jhi-translate.directive';
 import { JhiConfigService } from '../../src/config.service';
 
-function getElementHtml(element: ComponentFixture<JhiItemCountComponent>): string {
+function getElementHtml(
+    element: ComponentFixture<JhiItemCountComponent>
+): string {
     const res = element.nativeElement.querySelector('.jhi-item-count');
-    return (res && res.innerHTML) ? res.innerHTML.trim() : '';
-}
-
-function getElementAttribute(element: ComponentFixture<JhiItemCountComponent>, value: string): string {
```

Comment:
does it update correctly without this?

Reply:
I removed the `[attr.translateValues]="i18nValues()"` in the HTML, because the comment said it was here only for testing purposes, and I'm not a big fan of that.
I prefer testing directly the `i18Values()` function, that's why I don't use this method anymore.

Hope I understood your question 😄